sample-app/src/components/DropdownSection.tsx

```diff
@@ -24,17 +24,18 @@ export interface DropdownSectionProps {
 }
 
 export default function DropdownSection({
-  focusStatus,
+  isFocused = false,
   options,
   optionIdPrefix,
   onFocusChange = () => {},
   onLeaveSectionFocus = () => {},
-  onClickOption = () => {},
+  onSelect = () => {},
   label = '',
   cssClasses
 }: DropdownSectionProps): JSX.Element | null {
 
   const [focusedOptionIndex, setFocusedOptionIndex] = useState<number>(0);
+  const wasFocused = useRef<boolean>(isFocused);
 
   function incrementOptionFocus() {
```

Comment:
I was looking at the logic for `decrementOptionFocus`. In the `else` condition, we know that `newIndex <= -1`. That results in the following call `onLeaveSectionFocus(false)`. This call reads confusingly. It suggests that we don't leave the section's focus. That's outside the scope of this item, but I think we should fix before we release the sample-app.

Reply:
Would it be more clear to call it `onChangeSectionFocus`? Or does `onChangeSectionFocus(false)` still read as though the focus shouldn't be changed?

Reply:
We can figure it out! Like I said, outside the scope of this item.